testhelp/testhelper.go

```diff
@@ -187,3 +188,63 @@ func MergeMap(a, b map[string]any) map[string]any {
 	}
 	return c
 }
+
```

Comment:
as the majority of the code, this also existed already. Moved and added more documentation.